apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -139,6 +147,21 @@ export default class TeacherHomepage extends Component {
         {isEnglish && showSpecialAnnouncement && (
           <SpecialAnnouncementActionBlock/>
         )}
+        {announcements.length > 0 && (
+          <div>
+            <Notification
+              type={announcements[0].type || "bullhorn"}
```

Comment:
At some point we should think about reworking so we are passing one announcement rather than an array of announcements to the homepage.  I think this is leftover from an idea a long time ago that we might have a carousel of announcements to click through, but we've never done that.